llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -2617,6 +2617,13 @@ bool RISCVInstrInfo::verifyInstruction(const MachineInstr &MI,
     }
   }
 
+  if (int Idx = RISCVII::getFRMOpNum(Desc);
```

Comment:
This is checking the pseudo form, is there any way that we can add an additional assert which checks the pseudo against the underlying instruction?  The underlying instruction needs to have the use of FRM too doesn't it?

(Possible follow up)

Reply:
Wouldn't a disassembly use case be wrong if we didn't have the implicit use?

Reply:
It doesn't affect the printing of the instruction so the main use of disassembly wouldn't notice.